aries-site/src/data/structures/components.js

```diff
@@ -321,6 +321,7 @@ export const components = [
   },
   {
     name: 'DateInput',
+    accessibility: 'Passed WCAG 2.2 AAA',
```

Comment:
```suggestion
    accessibility: 'Failed WCAG 2.2 A',
```
I don't think we are passing WCAG A level